CHANGELOG.md

```diff
@@ -8,27 +8,45 @@
      users to define the fingerprint in the environment manually in order to
      adopt this behaviour, otherwise, by default, a new iteration will be
      created. This does not impact workflows where the fingerprint was defined
-     through the HCP_PACKER_ITERATION_FINGERPRINT environment variable, and
+     through the `HCP_PACKER_ITERATION_FINGERPRINT` environment variable, and
      these builds will work exactly as they did before.
      [GH-12172](https://github.com/hashicorp/packer/pull/12172)
 
+### PLUGINS
+* Remove provisioner plugins for Chef, Converge, Puppet, Salt, and Inspec as
```

Comment:
Is this in the 1.8.7 branch as well? If so we should probably add this under the release section

Reply:
This particular change is only in 1.9.0.

Reply:
Sounds good then, we can leave it here!